src/main/java/core/basesyntax/service/ReadingCsvFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadingCsvFile {
+    List<String> reading(String fileName);
```

Comment:
The method name 'reading' is a continuous tense verb, which is not a standard convention in method naming. Consider renaming it to 'read' or 'readFromFile' to align with common practices and to indicate the action more clearly.